src/containers/OrderReviewPage.jsx

```diff
@@ -174,31 +177,125 @@ class OrderReviewPage extends React.Component {
       'lineitems:element:dependentlineitems:element:item:definition',
       'lineitems:element:dependentlineitems:element:item:code',
     ];
-    const { orderData } = this.state;
-    const { history } = this.props;
-    const purchaseform = orderData._order[0]._purchaseform[0].links.find(link => link.rel === 'submitorderaction').uri;
-    login().then(() => {
-      cortexFetch(`${purchaseform}?followlocation=true&zoom=${purchaseZoomArray.sort().join()}`, {
-        method: 'post',
-        headers: {
-          'Content-Type': 'application/json',
-          Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthToken`),
-        },
-      })
-        .then(res => res.json())
-        .then((res) => {
-          this.setState({
-            isLoading: false,
-          });
-          this.giftCertificatesAddToCart();
-          this.trackTransactionAnalytics();
-          history.push('/purchaseReceipt', { data: res });
-        })
-        .catch((error) => {
-          // eslint-disable-next-line no-console
-          console.error(error.message);
+
+    if (orderData && orderData._order && orderData._order[0]._paymentmethodinfo && orderData._order[0]._billingaddressinfo) {
+      const purchaseform = orderData._order[0]._purchaseform[0].links.find(link => link.rel === 'submitorderaction').uri;
+      const uriForRequest = 'https://iojbbbi1o8.execute-api.us-east-2.amazonaws.com/test/sign';
```

Comment:
We'll make a call to the lambda to get this url, and sign the request to cybersource. This value should be externalized in `ep.config.json`.